### ISSUE TITLE
Rename everywhere to User -> Agent -> Session?

### ISSUE THREAD

**Issue by dehann** [JuliaRobotics/DistributedFactorGraphs.jl#862]
Drop name Robot in this case.
- User == Org.
- Agent is human or robot,
- Session is session.

Environment then involves one Org and many Agents / Sessions?

**Comment by GearsAD** [comment]
My take:

Orgs are groups of users - one level higher.

We can do an agent renaming, it would be quite invasive because RobotID is
used as part of the compound key in a lot of places.

**Comment by jim-hill-r** [comment]
I agree with sam. Users are individuals in the world (actual humans). Orgs are some entity like a company or school. Those orgs would provide permissions to users somehow. Or users can be added to orgs somehow.

**Comment by Affie** [comment]
closed by #1100
